tests/v0_tests/test_index.py

```diff
@@ -164,6 +154,7 @@ def run():
             return True
         assert run()
 
+    @mark.ignore_cloud_tests
     def test_create_cloud_index_non_default_param(self):
```

Comment:
we should probably test this for the cloud. But if there is an existing non-default-param index we are using for the cloud we make the test against that